client/cmd/dexc-desktop/pkg/publish-snap.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+source $(dirname "$0")/common.sh
+
+snapcraft login
+
```

Comment:
This will prompt for snapcraft account credentials? Is it expected that the app is already `snapcraft register`'d?

Reply:
Yes, it will ask for the account creds - the app must be registered in the same account.
https://snapcraft.io/snaps